### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by al-serebrov**
Hey!

Here are some changes connected with the styling of my superb flask application.
Added a couple jQuery event handlers (pretty simple, but they are my first steps in JavaScript).
Also added an url_for override in order to avoid browser caching of static resources (tbh the snippet is from stackoverflow).
Please let me know if you have any observations or notes.

Thank you in advance.

### CODE REVIEW

File: static/style.css [hunk 1]
```diff
@@ -1,20 +1,119 @@
...
+	width: 100%;
+}
+
+textarea {
+	width: 100%;
+}
+
+input[type="submit"], input[type="submit"]:focus, input[type="submit"]:active {
+    /* Remove all decorations to look like normal text */
+    height: 25px;
```
Comment (serebrov) [inline static/style.css:47]: The indentation is inconsistent, here there are 4 spaces, above - 8. I usually prefer using 4 spaces everywhere (and in js files too).

File: templates/layout.html [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <!doctupe html>
-<title>Flaskr</title>
-<link rel=stylesheet type=text/css href="{{ url_for('static', filename='style.css') }}?=q=1280549780">
+<head>
```
Comment (serebrov) [inline templates/layout.html]: `doctupe` above is mis-spelled

**Review by serebrov — APPROVED** [review]
LGTM, please check my (minor) comments.

File: templates/layout.html (hunk 2 above)
Comment (al-serebrov) [reply templates/layout.html]: wow, I was wondering why it's highlighted red :sweat_smile: 
corrected this

File: static/style.css (hunk 1 above)
Comment (al-serebrov) [reply static/style.css:47]: yeah, that's sublime text issue, I'll convert all indentations to spaces, it was tab-indented

**Comment by al-serebrov** [comment]
I also made changes in flash messages behaviour

**Pull request merged**